ts/parts/Utilities.ts

```diff
@@ -1283,7 +1283,7 @@ function isObject(obj: any, strict?: boolean): boolean {
  * @return {boolean}
  *         True if the argument is a HTML Element.
  */
-H.isDOMElement = function (obj: any): boolean {
+function isDOMElement(obj: any): boolean {
```

Comment:
Did you tried `function isDOMElement(obj: unknown): obj is HTMLElement`?

Reply:
No, will check that out 👍

Reply:
@bre1470 Please see latest changes

Reply:
Looks great! Thanks. 👍